reporter/src/test/kotlin/reporters/OpossumReporterTest.kt

```diff
@@ -0,0 +1,546 @@
+import org.ossreviewtoolkit.model.RepositoryProvenance
+import org.ossreviewtoolkit.model.ScanRecord
+import org.ossreviewtoolkit.model.ScanResult
+import org.ossreviewtoolkit.model.ScanSummary
+import org.ossreviewtoolkit.model.ScannerDetails
+import org.ossreviewtoolkit.model.ScannerRun
+import org.ossreviewtoolkit.model.Scope
+import org.ossreviewtoolkit.model.TextLocation
+import org.ossreviewtoolkit.model.VcsInfo
+import org.ossreviewtoolkit.model.VcsType
+import org.ossreviewtoolkit.model.config.AnalyzerConfiguration
+import org.ossreviewtoolkit.model.config.Excludes
+import org.ossreviewtoolkit.model.config.RepositoryConfiguration
+import org.ossreviewtoolkit.model.config.ScannerConfiguration
+import org.ossreviewtoolkit.model.config.ScopeExclude
+import org.ossreviewtoolkit.model.config.ScopeExcludeReason
+import org.ossreviewtoolkit.spdx.toSpdx
+import org.ossreviewtoolkit.utils.Environment
+
+class OpossumReporterTest : WordSpec({
+
+    "pathResolve()" should {
+        "work with basic cases" {
+            pathResolve("/") shouldBe "/"
+            pathResolve("/", "test") shouldBe "/test"
+            pathResolve("/", "/test") shouldBe "/test"
+            pathResolve("/", "/test/") shouldBe "/test/"
+            pathResolve("/tost", "test") shouldBe "/tost/test"
+            pathResolve("/tost", "/test") shouldBe "/tost/test"
+            pathResolve("/tost", "/test/") shouldBe "/tost/test/"
+        }
+
+        "work with lists" {
+            pathResolve(listOf("/", "test/to/path", "/something/else/")) shouldBe "/test/to/path/something/else/"
+        }
+    }
+    "convertToId()" should {
+        "work" {
+            pathResolve("/") shouldBe "/"
+            pathResolve("/", isDirFlag = true) shouldBe "/"
+            pathResolve("/", isDirFlag = false) shouldBe "/"
+            pathResolve("path/to/thing") shouldBe "/path/to/thing"
+            pathResolve("path/to/thing/") shouldBe "/path/to/thing/"
+            pathResolve("/path/to/thing/") shouldBe "/path/to/thing/"
+            pathResolve("path/to/thing/", isDirFlag = true) shouldBe "/path/to/thing/"
+            pathResolve("path/to/thing/", isDirFlag = false) shouldBe "/path/to/thing"
+            pathResolve("path/to/thing", isDirFlag = true) shouldBe "/path/to/thing/"
+            pathResolve("path/to/thing", isDirFlag = false) shouldBe "/path/to/thing"
+            pathResolve("/path/to/thing/", isDirFlag = true) shouldBe "/path/to/thing/"
+            pathResolve("/path/to/thing/", isDirFlag = false) shouldBe "/path/to/thing"
+            pathResolve("/path/to/thing", isDirFlag = true) shouldBe "/path/to/thing/"
+            pathResolve("/path/to/thing", isDirFlag = false) shouldBe "/path/to/thing"
+        }
+    }
+
+    "generateOpossumInput()" should {
+        val result = createOrtResult()
+        val opossumInput = OpossumReporter().generateOpossumInput(result, emptySet())
+
+        "should be somehow valid" {
```

Comment:
Make sure that all test names make sense together with `"generateOpossumInput()` should"`. For example, in this case "~~should~~ be valid".